src/handler/global/watcher.ts

```diff
@@ -0,0 +1,228 @@
+import { Client, User } from "discord.js";
+import { getCustomRepository } from "typeorm";
+
+import { ScheduledJob } from "../../decorators";
+
+import {
+    UserSettingsRepository,
+    Timers,
+    Counts,
+} from "../../repository/UserSettingsRepository";
+import { UserSettings } from "../../entity/UserSettings";
+import * as wrapper from "../../wrapper/wrapper";
+import { makeLogger } from "../../logger";
+import { NotificationData } from "../../wrapper/models/notification";
+
+const logger = makeLogger(module);
+
+function getRepository(): UserSettingsRepository {
+    return getCustomRepository(UserSettingsRepository);
+}
+
+export class Watcher {
```

Comment:
not sure this is the right directory for this file... its not a command so it doesnt really fall into either the global or guild category

Reply:
It needs to be in src/handler so that it's loaded when the bot launches. We could add another sub directory for non-command handlers